src/vs/workbench/services/textfile/common/textFileEditorModelManager.ts

```diff
@@ -158,7 +158,7 @@ export class TextFileEditorModelManager extends Disposable implements ITextFileE
 
 			// Install state change listener
 			this.mapResourceToStateChangeListener.set(resource, model.onDidStateChange(state => {
-				const event = new TextFileModelChangeEvent(model, state);
+				const event = new TextFileModelChangeEvent(model!, state);
```

Comment:
@mjbvz how come this is needed when `model` is being created a few lines above? Sounds like a bug in TS to me?